src/backend/vulkan/Cargo.toml

```diff
@@ -30,6 +30,7 @@ parking_lot = "0.11"
 smallvec = "1.0"
 raw-window-handle = "0.3"
 inplace_it = "0.3.3"
+renderdoc = { version = "0.10.1", optional = true }
```

Comment:
can we depend on renderdoc-sys instead of renderdoc here?

Reply:
Should be technically possible, but it would mean probably reimplementing a good bit of the unsafe code in the renderdoc crate, for loading the lib and getting the function pointers
cf. https://github.com/ebkalderon/renderdoc-rs/blob/master/src/version.rs#L79

What would the advantages be of using renderdoc-sys instead of renderdoc?

Reply:
I think it would make sense for us to fully control how external libraries are loaded. For example `renderdoc` crate uses
```
static RD_LIB: OnceCell<Library> = OnceCell::new();
```
but here we'd want to just own the library inside our `struct Instance`, just how we own the Vulkan DLL and other DLLs.

Overall, we want minimal dependencies and maximum control. With renderdoc-sys, we may have it as non-optional, since we'd be in full control of how the API is used.

Reply:
Good point!